sys/net/gnrc/network_layer/sixlowpan/gnrc_sixlowpan.c

```diff
@@ -200,7 +199,11 @@ static void _receive(gnrc_pktsnip_t *pkt)
         }
 
         pkt = gnrc_pktbuf_remove_snip(pkt, sixlowpan);
+#ifdef MODULE_CCN_LITE
+        payload->type = GNRC_NETTYPE_CCN;
+#else /* MODULE_CCN_LITE */
```

Comment:
```suggestion
#elif defined(MODULE_GNRC_IPV6)
```

(and then `UNDEF` for the else branch)